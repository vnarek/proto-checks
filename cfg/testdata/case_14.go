package main

func main() {
	*a = 5
	*b = *c + *d
	*e = foo(5, *f, 7)
}