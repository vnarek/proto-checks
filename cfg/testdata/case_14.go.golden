[START]
  [*a]
    [*c]
      [*d]
        [*b]
          [_t3 = alloc]
            [*f]
              [*e = _t3]
